stream/pbf.js

```diff
@@ -0,0 +1,68 @@
+
+/**
+  The pbf parser is responsible for configuring and executing a pbf parser and
+  returning a valid readable stream.
+**/
+
+var fs = require('fs'),
+    pbf2json = require('pbf2json'),
+    settings = require('pelias-config').generate(),
+    features = require('../config/features');
+
+// Create a new pbf parser stream
+function createPbfStream( opts ){
```

Comment:
Indeed, doesn't have to be in JSDoc format, but an enumeration of all the properties would be great.